server_db/controllers/accountController.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable consistent-return */
+import createAccount from '../models/accountQuery';
+import db from '../models/db';
+
+class AccountController {
+  static async postAccount(req, res) {
```

Comment:
You did not add doc-string to your methods
All method should contain doc-string describing what the method is doing,
the arguments/params and the return value

Reply:
Ok